.github/issue_template.md

```diff
@@ -1,48 +1,30 @@
-Note: This is just a template, so feel free to use/remove the unnecessary things
-
 ### Description
-- Type: Bug | Enhancement | Question
-- Related issue: `#abc`
-- Priority: Blocker | Major | Minor
-
----------------------------------------------------------------
-## Bug
-
-**Target**
-K64F|??
-
-**Toolchain:**
-GCC_ARM|ARM|IAR
-
-**Toolchain version:**
-
-**mbed-cli version:** 
-(`mbed --version`)
-
-**mbed-os sha:**
-(`git log -n1 --oneline`)
-
-**DAPLink version:**
-
-**Expected behavior**
-
-**Actual behavior**
-
-**Steps to reproduce**
-
-----------------------------------------------------------------
-## Enhancement
-
-**Reason to enhance or problem with existing solution**
-
-**Suggested enhancement**
-
-**Pros**
-
-**Cons**
-
------------------------------------------------------------------
-
-## Question
+<!-- 
+    Required
+    Add detailed description of what you are reporting.
+    Good example: https://os.mbed.com/docs/latest/reference/workflow.html
+    Things to consider sharing:
+    - What target does this relate to?
+    - What toolchain are you using?
```

Comment:
Maybe tweak to say:
- What tools and their versions are being used?
- Which compiler and version?

Reply:
Updated via a new commit